starter-project-backend-g31/src/controllers/rating.ts

```diff
@@ -72,6 +81,10 @@ export const updateRating = async (req: any, res: any) => {
   try {
     const rating = await Rating.findById({ _id: req.params.id })
     if (!rating) return res.status(404).send("Rating not found to be updated")
```

Comment:
Change response message since this would be shown to users on the frontend. like.. "Rating not found!"